src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
 
+        Machine[] machines = new Machine[]{excavator,bulldozer,truck};
+
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
```

Comment:
Similarly to the 'doWork' method, the 'stopWork' method should also provide an informative message indicating which Machine has stopped working.